doc/development/plugins.rst

```diff
@@ -470,12 +470,15 @@ pieces of functionality:
 Note that these properties are only applicable to devices that provided derivatives or VJPs. If your device
 does not provide derivatives, you can safely ignore these properties.
 
-The workflow options are ``use_device_gradient``, ``use_device_jacobian_product``, and ``grad_on_execution``. 
+The workflow options are ``use_device_gradient``, ``use_device_jacobian_product``, ``grad_on_execution``,
+and ``convert_to_numpy``. 
 ``use_device_gradient=True`` indicates that workflow should request derivatives from the device. 
 ``grad_on_execution=True`` indicates a preference to use ``execute_and_compute_derivatives`` instead
 of ``execute`` followed by ``compute_derivatives``. Finally, ``use_device_jacobian_product`` indicates
```

Comment:
Since now this section also talks about `convert_to_numpy`
```suggestion
of ``execute`` followed by ``compute_derivatives``. ``use_device_jacobian_product`` indicates
```